deploy/osd-machine-api/010-machine-api.MachineHealthCheck.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineHealthCheck
+metadata:
+  name: srep-healthcheck
+  namespace: openshift-machine-api
+spec:
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-machine-role: worker
+      machine.openshift.io/cluster-api-machine-role: infra
+  unhealthyConditions:
+  - type:    "Ready"
+    timeout: "480s"
+    status: "False"
+  - type:    "Ready"
+    timeout: "480s"
+    status: "Unknown"
+  maxUnhealthy: "40%"
```

Comment:
This indicates that if 40% of the machines go unhealthy then machine-api(-operator?) will stop taking action on the assumption that human intervention is needed at that point.